addon/components/challenges/add.js

```diff
@@ -54,10 +115,21 @@ export default Ember.Component.extend(ClickOutside, {
     saveDismiss(){
       this.actions.save.bind(this)();
       this.actions.dismiss.bind(this)();
+    },
+    selectMajor(){
```

Comment:
Why not use toggle inputs of some kind and have the values be `major` and `minor`? That way, the toggling is handled by the browser, and you just have to get the value of the input.